src/main/kotlin/org/jetbrains/research/libsl/visitors/BlockStatementVisitor.kt

```diff
@@ -5,22 +5,25 @@ import org.jetbrains.research.libsl.context.FunctionContext
 import org.jetbrains.research.libsl.nodes.*
 
 class BlockStatementVisitor(
-    private val functionContext: FunctionContext,
-    private val statements: MutableList<Statement>
-    ) : LibSLParserVisitor<Unit>(functionContext) {
+    private val functionContext: FunctionContext
+) : LibSLParserVisitor<Unit>(functionContext) {
+    val statements: MutableList<Statement> = mutableListOf()
 
     override fun visitExpression(ctx: LibSLParser.ExpressionContext) {
         val expressionVisitor = ExpressionVisitor(functionContext)
         val expression = ExpressionStatement(expressionVisitor.visitExpression(ctx))
-        println(expression)
         statements.add(expression)
     }
 
     override fun visitVariableAssignment(ctx: LibSLParser.VariableAssignmentContext) {
         val expressionVisitor = ExpressionVisitor(functionContext)
         val left = expressionVisitor.visitQualifiedAccess(ctx.qualifiedAccess())
         val op = AssignOps.fromString(ctx.op.text)
-        val value = expressionVisitor.visitExpression(ctx.expression())
+        val value = if (ctx.assignmentRight() != null) {
```

Comment:
I'd reuse code from `ExpressionVisitor` class here (so, this code will be simplified to `value = ctx.assignmentRight()?.let { expressionVisitor.visitAssignmentRight(it) }`)